src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/bulkscan/transformation/output/SuccessfulTransformationResponse.java

```diff
@@ -4,23 +4,17 @@
 import lombok.Builder;
 import lombok.Data;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Data
 @Builder
 public class SuccessfulTransformationResponse {
```

Comment:
->lib

Reply:
Moved into the BSP Commons Lib as requested